frontend/src/pages/Player/RightPlayerPanel/components/NetworkResourcePanel/NetworkResourceLogs.tsx

```diff
@@ -67,6 +67,7 @@ export const NetworkResourceLogs: React.FC<{
 		logCursor: undefined,
 		startDate,
 		endDate,
+		disablePolling: true,
```

Comment:
Turned off polling here - I don't think we would ever want it with the set dates?